interfaces/portal/package.json

```diff
@@ -92,15 +92,15 @@
     "eslint-plugin-regexp": "^2.9.0",
     "eslint-plugin-simple-import-sort": "^12.1.1",
     "eslint-plugin-sort-class-members": "^1.21.0",
-    "globals": "^16.2.0",
-    "jasmine-core": "~5.8.0",
+    "globals": "^16.3.0",
+    "jasmine-core": "~5.10.0",
```

Comment:
Maybe, we need this:

```json
  "overrides": {
    "karma-jasmine": {
      "jasmine-core": "^4.1.0 || ^5.0.0"
    }
  }
```

Added at the bottom of this file, next to the `ngx-filesize`-override...

Maybe that helps the tests run?

Reply:
Fixed via incoming PR